js/views/itemListVw.js

```diff
@@ -10,6 +10,7 @@ module.exports = Backbone.View.extend({
   initialize: function(options){
     var self = this;
     this.options = options || {};
+    this.category = options.category;
```

Comment:
Here it should be:

this.category = options.category || "all";

just in case no category is passed in.